File: underwriting-service/src/foam.kt
package com.affirm.underwriting

import com.foam.kotlin.Foam
import com.affirm.underwriting.config.Config

fun initFoam(config: Config) {
    Foam.init(
        serviceName = "underwriting-service",
        isProduction = config.isProduction,
        apiKey = config.foamApiKey,
        environment = if (config.isProduction) "production" else "staging",
        tags = mapOf("team" to "credit-engineering", "domain" to "decisioning")
    )
}

File: checkout-api/src/foam.py
import foam
from config import FOAM_API_KEY, IS_PRODUCTION

foam.init(
    service_name="checkout-api",
    is_production=IS_PRODUCTION,
    api_key=FOAM_API_KEY,
    environment="production" if IS_PRODUCTION else "development",
    tags={"team": "merchant-integrations", "domain": "payments"}
)